polyglot/piranha/src/lib.rs

```diff
@@ -346,7 +379,7 @@ impl FlagCleaner {
     loop {
       let current_rules = self.rule_store.global_rules();
 
-      info!("{}", format!("# Global rules {}", current_rules.len()));
+      debug!("\n{}", format!("# Global rules {}", current_rules.len()));
```

Comment:
Again, not sure how the API works here... but is it necessary to use `format` rather than just `debug!(...)` itself expanding a format string? Similar question on a few cases below.

Reply:
Yes, here using the `format!` macro is unnecessary.